sfepy/discrete/common/domain.py

```diff
@@ -8,19 +8,23 @@
                                           sort_by_dependency, get_parents)
 from sfepy.discrete.parse_regions import create_bnf, visit_stack, ParseException
 
-def region_leaf(domain, regions, rdef, functions):
+
+def region_leaf(domain, regions, rdef, functions, tdim):
     """
     Create/setup a region instance according to rdef.
     """
     n_coor = domain.shape.n_nod
     dim = domain.shape.dim
+    cmesh = domain.cmesh_tdim[tdim]
 
     def _region_leaf(level, op):
         token, details = op['token'], op['orig']
 
         if token != 'KW_Region':
             parse_def = token + '<' + ' '.join(details) + '>'
-            region = Region('leaf', rdef, domain, parse_def=parse_def)
+            if token != 'KW_COG':
```

Comment:
Is this used/defined anywhere? I cannot find it.

Reply:
There should be `E_COG` instead of `KW_COG`.